doc/api/vm.md

```diff
@@ -866,6 +866,36 @@ local scope, so the value `localVar` is changed. In this way
 `vm.runInThisContext()` is much like an [indirect `eval()` call][], e.g.
 `(0,eval)('code')`.
 
+## vm.compileFunction
+<!-- YAML
+added: REPLACEME
+-->
+* `code` {string} The JavaScript code to compile and run.
+* `params` {Array|undefined} An array of string containing all paramters for
```

Comment:
We don't explicitly put `undefined` in the types, because it is implied from the function signature. Instead, describe the default value here (which I think is the empty array).